### PR TITLE
Add Android Studio support

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
Adds the Android Studio everywhere and sorts tabs always in the same order (Android Studio / IntellJ IDEA, then Eclipse, then CLI).

https://docs.microej.com/en/feature-android-studio/SDK6UserGuide

### CODE REVIEW

File: SDK6UserGuide/install.rst [hunk 1]
```diff
@@ -108,12 +109,12 @@ Once your favorite IDE is installed, the MicroEJ plugin must be installed.
 
 .. tabs::
 
-   .. tab:: IntelliJ IDEA
+   .. tab:: Android Studio / IntelliJ IDEA
```
Comment (denisbedin) [inline SDK6UserGuide/install.rst:112]: The description of the images in the tab must be updated to add Android Studio.

**Review by denisbedin — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/createProject.rst [hunk 2]
```diff
@@ -150,46 +278,41 @@ This chapter explains the different ways to create a new project.
       You can then learn :ref:`how to launch the build of the project <sdk_6_build_project>`, 
       or :ref:`how to run it on the Simulator <sdk_6_run_on_simulator>` in the case of an Application.
 
-   .. tab:: IntelliJ IDEA
+   .. tab:: Command Line Interface
```
Comment (bguedas) [inline SDK6UserGuide/createProject.rst:260]: The tab for CLI is named "Command Line Interface" here, but "CLI" everywhere else. I think that "Command Line Interface" is better

File: SDK6UserGuide/testProject.rst [hunk 3]
```diff
@@ -149,20 +149,20 @@ Once the testsuite is configured, it can be run thanks to the ``test`` Gradle ta
 This task is bound to the ``check`` and the ``build`` Gradle lifecycle tasks,
 which means that the tests are also executed when launching one of these tasks.
    
-In order to execute the testsuite from Eclipse and IntelliJ IDEA, double-click on the task in the Gradle tasks view:
+In order to execute the testsuite from Android Studio, IntelliJ IDEA or Eclipse, double-click on the task in the Gradle tasks view:
 
 .. tabs::
```
Comment (bguedas) [inline SDK6UserGuide/testProject.rst:154]: We should have a tab for CLI with the content above on how to run the task (`./gradlew test`)

**Review by bguedas — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/install.rst [hunk 4]
```diff
@@ -94,12 +94,13 @@ Install the IDE
 Using an IDE is highly recommended for developing MicroEJ projects, making the development more comfortable and increasing productivity.
 The two following IDEs are supported: 
 
+- `Android Studio <https://developer.android.com/studio>` - Minimum supported version is ``Bumblebee - 2021.1.1``.
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/install.rst]: Done.

File: SDK6UserGuide/install.rst [hunk 5]
```diff
@@ -94,12 +94,13 @@ Install the IDE
 Using an IDE is highly recommended for developing MicroEJ projects, making the development more comfortable and increasing productivity.
 The two following IDEs are supported:
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/install.rst]: Done.

File: SDK6UserGuide/install.rst [hunk 6]
```diff
@@ -108,12 +109,12 @@ Once your favorite IDE is installed, the MicroEJ plugin must be installed.
 
 .. tabs::
 
-   .. tab:: IntelliJ IDEA
+   .. tab:: Android Studio / IntelliJ IDEA
 
       Follow these steps to install the latest stable version of the MicroEJ plugin for IntelliJ IDEA:
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/install.rst]: Done.

File: SDK6UserGuide/createProject.rst [hunk 7]
```diff
@@ -7,47 +7,147 @@ This chapter explains the different ways to create a new project.
...
+      - Declare the dependencies required by your project in the ``dependencies`` block. For example::
+      
+            dependencies {
+                implementation("ej.api:edc:1.3.5")
+            }
       
       .. note::
-        These steps are the ones proposed when creating a project with Gradle ``8.2.1``. 
-        Depending on the Gradle version used, the steps to create a project can be slightly different.
+         By default, Android Studio and IntelliJ IDEA automatically save any file change,
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/createProject.rst]: Done.

File: SDK6UserGuide/install.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/install.rst:112]: Done.

File: SDK6UserGuide/createProject.rst (hunk 2 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/createProject.rst:260]: Agreed, "Command Line Interface" used everywhere.

File: SDK6UserGuide/testProject.rst (hunk 3 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/testProject.rst:149]: Done.

**Review by denisbedin — APPROVED** [review]

**Review by bguedas — APPROVED** [review]

**Pull request merged**